docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,324 @@
+
+The above steps should only occur once per life cycle of SDL Core
+
+!!!
+
+## Communicating with SDL Core
+
+This section describes the message structure for communication between your HMI and SDL Core. The JSON RPC 2.0 format is taken as a basis.
+
+From this point forward the actors for exchanging messages will be considered:
+  - **Client** - can send requests and notifications
+  - **Server** - can provide responses to requests from a Client and send notifications
+
+## Request
+An RPC call is represented by sending a Request object to a Server. The Request object has the following properties
+
+| Property | Description    |
+| :------------- | :------------- |
+| "id"       | An identifier established by the Client. This value must be of unsigned int type in the frames of communication between your HMI and SDL Core. The value should never be null. If "id" is not included the message is assumed to be a notification and the receiver should not respond.|
+| "jsonrpc" | A string specifying the version of JSON RPC protocol being used. Must be exactly **"2.0"** currently in all versions of SDL Core.|
+| "method" | A String containing the information of the method to be invoked. The format is `[componentName].[methodName]`.|
+| "params" | A structured object that holds the parameter values to be used during the invocation of the method. This property may be omitted.|
+
+### Example Requests
+#### Request with no Parameters
+```json
+{
+  "id": 125,
+  "jsonrpc": "2.0",
+  "method": "Buttons.GetCapabilities"
+}
+```
+#### Request with Parameters
+```json
+{
+  "id": 92,
+  "jsonrpc": "2.0",
+  "method": "UI.Alert",
+  "params": {
+    "alertStrings": [
+      {
+        "fieldName": "alertText1",
+        "fieldText": "WARNING"
+      },
+      {
+        "fieldName": "alertText2",
+        "fieldText": "Adverse Weather Conditions Ahead"
+      }
+    ],
+    "duration": 4000,
+    "softButtons": [
+      {
+        "type": "TEXT",
+        "text": "OK",
+        "softButtonID": 697,
+        "systemAction": "STEAL_FOCUS"
+      }
+    ],
+    "appID": 8218
+  }
+}
+```
+
+## Notification
+A notification is a request object without an `id` property. For all the other properties, see the Request Section above.
+
+The receiver should not reply to a notification, i.e. no response object needs to be returned to the client upon receipt of a notification.
+
+### Example Notifications
+#### Notification with no Parameters
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "UI.OnReady"
+}
+```
+#### Notifications with Parameters
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "BasicCommunication.OnAppActivated",
+  "params": {
+    "appID": 6578
+  }
+}
+
+{
+  "jsonrpc": "2.0",
+  "method": "Buttons.OnButtonPress",
+  "params": {
+    "mode": "SHORT",
+    "name": "OK"
+  }
+}
+```
+
+## Response
+On receipt of a request message, the server must reply with a response. The response is expressed as a single JSON Object with the following properties.
```

Comment:
```suggestion
On receipt of a request message, the server must reply with a Response. The Response is expressed as a single JSON Object with the following properties.
```

Since request is capitalized above, maybe response should be here.